### PR TITLE
Added the navigation bar from the lecture

### PR DESCRIPTION
**Pull request by Faberen**

### CODE REVIEW

**Comment by SkarpKat** [comment]
Smukt

**Review by SkarpKat — APPROVED** [review]

**Review by Lukan707 — CHANGES_REQUESTED** [review]
Nice work.
Can we perphaps take care of the warning?

**Comment by Faberen** [comment]
> Nice work. Can we perphaps take care of the warning?

Fixed it in latest commit

**Review by annahj1006 — APPROVED** [review]

File: src/Chirp.Razor/Pages/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <div class=page>
     <h1><a class="logoStyling" href="~/"><img src="/images/icon1.png" alt="Icon1"/>Chirp!</a></h1>
         <div class=navigation>
-        @if (User.Identity.IsAuthenticated)
+        @if (User?.Identity?.IsAuthenticated == true)
         {
```
Comment (Lukan707) [inline src/Chirp.Razor/Pages/Shared/_Layout.cshtml:13]: :)

**Review by Lukan707 — APPROVED** [review]
: )

**Pull request merged**